extension/oidcauthextension/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/extension/oidcauthextension
+
+go 1.16
+
+require (
+	github.com/coreos/go-oidc v2.2.1+incompatible
+	github.com/stretchr/testify v1.7.0
+	go.opentelemetry.io/collector v0.33.0
```

Comment:
Should this depend on the same collector version the rest of the repo is pinned to?

Reply:
yes, I will change this